SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -325,7 +386,7 @@ void SSpatialOutputLog::FormatAndPrintRawLogLine(const FString& LogLine)
 		}
 	}
 
-	ELogVerbosity::Type LogVerbosity = ELogVerbosity::Display;
+	ELogVerbosity::Type LogVerbosity;
```

Comment:
How come this was removed? AFAIK this is required

Reply:
Nah, it's guaranteed to be set in the switch so the initialization is unnecessary. Rider also flagged this. The compiler would probably remove it anyway.

Reply:
yeah but isn't the default for `ELogVerbosity` `NoLogging` which means we would drop and lose those logs?

Reply:
Not sure what you mean. `LogVerbosity` will always get an explicit verbosity set since there is a catch-call `else` below. There is no mention of `NoLogging` anywhere.